File: Native/Test/Fabric/node-add-member-default-data.js
FABRIC = require('Fabric').createClient();

var node = FABRIC.DG.createNode("node");
node.addMember("foo","Scalar",3.141);
console.log(node.getData("foo", 0));
node.setData("foo", 0, 2.718);
console.log(node.getData("foo", 0));
node.setCount(2);
console.log(node.getData("foo", 1));

FABRIC.flush();

File: Native/Test/Python/node-add-member-default-data.py
#
#  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
#

import fabric
fabricClient = fabric.createClient()

node = fabricClient.DG.createNode("node")
node.addMember("foo","Scalar",3.141)
print(node.getData("foo", 0))
node.setData("foo", 0, 2.718)
print(node.getData("foo", 0))
node.resize(2)
print(node.getData("foo", 1))

fabricClient.close()
